### PR TITLE
Making prometheus worse for testing comparisons in parca

### PR DESCRIPTION
**Pull request by Adib234**

### CODE REVIEW

File: cmd/prometheus/main.go [hunk 1]
```diff
@@ -607,7 +607,7 @@ func main() {
 	prometheus.MustRegister(configSuccess)
 	prometheus.MustRegister(configSuccessTime)
 
-	b := make([]byte, 50e6) // ~50 MB
+	b := make([]byte, 5000e6) // ~50 MB
```
Comment (bernot-dev) [inline cmd/prometheus/main.go]: `5000e6` == `5e9` == 5 GB

Comment (Adib234) [reply cmd/prometheus/main.go]: yes thats intentional, on parca the `main` branch and pr #188 seemed to similar in memory consumption

**Pull request closed without merging**